### PR TITLE
Update Teleste_HDP301_TSEMP.md

### PR DESCRIPTION
**Pull request by SaraGlu**
Added data about new range

### CODE REVIEW

File: connector/doc/Teleste_HDP301_TSEMP.md [hunk 1]
```diff
@@ -15,21 +15,24 @@ A **serial** connection is used in order to successfully retrieve and configure
 | Range              | Key Features      | Based on | System Impact  |
 |--------------------|-------------------|----------|----------------|
 | 1.0.0.x [Obsolete] | Initial version.  | -        | -              |
-| 2.0.0.x [SLC Main] | Full refactoring. | -        | Full refactor. |
+| 2.0.0.x            | Full refactoring. | -        | Full refactor. |
```
Comment (MariekeGO) [inline connector/doc/Teleste_HDP301_TSEMP.md:18]: The "System Impact" column is supposed to inform the user of how the changes will affect their system, so could you adjust this to make this more clear? Will previous alarm monitoring/trending for the element be lost, will the element need to be re-made, ... ?

Comment (SaraGlu) [reply connector/doc/Teleste_HDP301_TSEMP.md]: No problem, I just added the missing information.

Comment (MariekeGO) [reply connector/doc/Teleste_HDP301_TSEMP.md]: Should the alarm and trend templates also be remade for range 2.0.0.x? The "full refactor" description there isn't very helpful for the user.

Comment (SaraGlu) [reply connector/doc/Teleste_HDP301_TSEMP.md]: Yes, I added missing information.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**